plugins/kubernetes/app/controllers/kubernetes_project_controller.rb

```diff
@@ -1,16 +1,15 @@
 class KubernetesProjectController < ApplicationController
-  before_action :authorize_deployer!
-  before_action :project
+  include ProjectLevelAuthorization
+  before_action :authorize_project_deployer!
```

Comment:
I'm wondering if the viewer should be able to see the dashboard we'll be making?